main.py

```diff
@@ -51,6 +50,35 @@ async def get_book_by_isbn(isbn: str) -> BookDto:
     book.isbn = isbn
     return book
 
+@app.post("/book", response_description="201")
+async def create_book(book: BookDto):
+    # Create a new book record for testing
+    new_book = BookEntity.create(
+        isbn="9783161484100",
+        title="Super Derp!!",
+        creators="Author Name",
+        copyright_date="2021-01-01",
+        summary="Awesome badassery adventures",
+        genres="Fiction",
+        form="Paperback",
+        format="Standard",
+        pages="350",
+        type="Novel",
+        publisher="Example Publisher",
+        publication_date="2021-05-20",
+        awards="Best Book Award",
+        reading_level="Advanced",
+        banned_book=False,
+        topics="Adventure, Exploration",
+        subjects="Literature",
+        target_audience="Adults",
+        alternate_titles="The Example Book"
+    )
+    print(f"Book created: {new_book.title} with ID {new_book.book_id}")
+    book_entity = BookMapper.dto_to_entity(book)
+    book_entity.save()
```

Comment:
A bit confused here. Which line is actually adding the book here, `BookEntity.create()` or `book_entity.save()`?

Reply:
Turns out they are both used for persisting data. The difference between them is very subtle as seen [here](https://docs.peewee-orm.com/en/latest/peewee/quickstart.html#storing-data)